app/graphql/mutations/item_create.rb

```diff
@@ -8,7 +8,7 @@ class ItemCreate < BaseMutation
 
     argument :trip_id, Integer, required: true
     argument :item_name, String, required: true
-    argument :category, Integer, required: true
+    argument :category, String, required: false
```

Comment:
Thanks for changing this and helping out the FE, but I believe this category argument needs to be deleted.  That way if an item is created for the category "essentials" for example, that item isn't added to the essentials list for other new trips created.

Reply:
Yea you are absolutely right we should remove that argument all together.